shortfin/src/CMakeLists.txt

```
@@ -20,10 +20,15 @@ message(STATUS "Linking optional components '${_SHORTFIN_LIB_OPTIONAL_COMPONENTS
 shortfin_public_library(
   NAME
     shortfin
+  LINUX_LD_SCRIPT ${CMAKE_CURRENT_SOURCE_DIR}/shortfin.ld
```

```suggestion
  LINUX_LD_SCRIPT ${libshortfin_SOURCE_DIR}/build_toold/shortfin.ld
```
Should we move `shortfin/src/shortfin.ld` to `shortfin/build_toold/shortfin.ld`?

---

There will eventually be multiple shared libraries, and the linker scripts are source code, so I think they live with the library.